Packages/doc/epoch_information.rst

```diff
@@ -0,0 +1,119 @@
+.. _epoch_information_doc:
+
+=================
+Epoch Information
+=================
+
+Description
+-----------
+
+Whenever a stimset is used to acquire data MIES generates also information about epochs.
```

Comment:
How about: At acquisition time, sweep epoch metadata is generated.

Reply:
I changed it.